server/repository/src/migrations/mod.rs

```diff
@@ -89,6 +90,7 @@ pub fn migrate(
         Box::new(v1_05_00::V1_05_00),
         Box::new(v1_05_04::V1_05_04),
         Box::new(v1_06_00::V1_06_00),
+        Box::new(v1_07_00::V1_07_00),
```

Comment:
Added v1_07_00 translations to pack variant, and moved all pack_variant translations there